Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ruby:2.7.6
```

Comment:
Use ruby 3.1 alpine image

Reply:
@tachyons I tried using ruby alpine images but it is giving errors while building the image!
So far, the above version works fine!
WDYT?